tests/pos/i17525.scala

```diff
@@ -0,0 +1,4 @@
+object Extract {
+  transparent inline def unapply(value: String): Option[Tuple] = Some((1, "two"))
+}
+def fail(): Unit = "" match { case Extract(a, b) => }
```

Comment:
can you make the test do stuff in the case based on the transparent type?

Reply:
Done